src/components/NewsletterForm/fetch-newsletter-response.ts

```diff
@@ -1,11 +1,15 @@
 export const fetchNewsletterResponse = async (
+  email: string,
   endpointUrl: string,
-  email: string
+  researchOption?: string,
+  type?: string
 ) => {
   const requestOptions = {
     method: 'POST',
     body: JSON.stringify({
-      email
+      email,
+      research_dd:researchOption,
```

Comment:
should this now be research_options?

Reply:
I think for consistency researchOption is good as we are doing camel case in props.